src/commands/api-mesh/__tests__/log-get.test.js

```diff
@@ -0,0 +1,186 @@
+		parseSpy.mockResolvedValue({
+			args: { rayId: 'ray1' },
+			flags: {
+				ignoreCache: mockIgnoreCacheFlag,
+			},
+		});
+		getMeshId.mockResolvedValue('12345');
+		getLogsByRayId.mockResolvedValue({
+			'EventTimestampMs': 123456789,
+			'Exceptions': 'None',
+			'Logs': 'Log data',
+			'Outcome': 'Success',
+			'meshId': 'mesh1',
+			'rayId': 'ray1',
+			'URL': 'http://example.com',
+			'Request Method': 'GET',
+			'Response Status': 200,
+		});
+	});
+
+	afterEach(() => {
+		platformSpy.mockRestore();
+	});
+
+	test('snapshot FetchLogsCommand', () => {
+		expect(FetchLogsCommand.description).toMatchInlineSnapshot(
+			`"Get the Log of a given mesh by RayId"`,
+		);
+		expect(FetchLogsCommand.args).toMatchInlineSnapshot(`
+		        [
+		          {
+		            "description": "to fetch the log ",
+		            "name": "rayId",
+		            "required": true,
+		          },
+		        ]
+	    `);
+		expect(FetchLogsCommand.flags).toMatchInlineSnapshot(`
+		                {
+		                  "ignoreCache": {
+		                    "allowNo": false,
+		                    "char": "i",
+		                    "default": false,
+		                    "description": "Ignore cache and force manual org -> project -> workspace selection",
+		                    "parse": [Function],
+		                    "type": "boolean",
+		                  },
+		                }
+	        `);
+		expect(FetchLogsCommand.aliases).toMatchInlineSnapshot(`[]`);
+	});
+});
+
+test('should handle log not found error', async () => {
+	getLogsByRayId.mockRejectedValue(new Error('LogNotFound'));
+
+	const runResult = FetchLogsCommand.run();
+
+	return runResult.catch(err => {
+		expect(err.message).toMatchInlineSnapshot(`
+		"No logs found for RayID ray1. Check the RayID and try again. RequestId: dummy_request_id. Alternatively, you can use the following command to get all logs for a 30 minute time period: 
+		aio api-mesh log-get-bulk --startTime YYYY-MM-DDTHH:MM:SSZ --endTime YYYY-MM-DDTHH:MM:SSZ --filename mesh_logs.csv"
+	`);
+		expect(logSpy.mock.calls).toMatchInlineSnapshot(`[]`);
+		expect(errorLogSpy.mock.calls).toMatchInlineSnapshot(`
+		[
+		  [
+		    "No logs found for RayID ray1. Check the RayID and try again. RequestId: dummy_request_id. Alternatively, you can use the following command to get all logs for a 30 minute time period: 
+		aio api-mesh log-get-bulk --startTime YYYY-MM-DDTHH:MM:SSZ --endTime YYYY-MM-DDTHH:MM:SSZ --filename mesh_logs.csv",
+		  ],
+		]
+	`);
+	});
+});
+
+test('should handle server error', async () => {
+	getLogsByRayId.mockRejectedValue(new Error('Internal Server Error. Please try again later.'));
+
+	const runResult = FetchLogsCommand.run();
+
+	return runResult.catch(err => {
+		expect(err.message).toMatchInlineSnapshot(
+			`"Unable to get mesh logs. Please check the details and try again. If the error persists please contact support. RequestId: dummy_request_id"`,
```

Comment:
Shouldn't this match with what we get [here](https://github.com/adobe/aio-cli-plugin-api-mesh/pull/167/files#diff-5205134531b36e1e4ebee9805763ec4205237afe58021924bb2d6fdc26399293R69-R71)?
`Server error while fetching logs for RayId ${rayId}. Please try again later. RequestId: ${global.requestId}`,`

Reply:
yeah needs to be updated